src/std/os/fdio.ss

```diff
@@ -43,6 +43,21 @@
     (do-retry-nonblock (_close raw)
       (close raw))))
 
+(def (flush raw)
+  (if (fd? raw)
+    (check-os-error (_flush raw)
+      (flush raw))
+    (force-output raw 2)))
```

Comment:
that branch probably unnecessary

Reply:
It's necessary because `raw` might be a port, the branches are in the incorrect order though and I need to wrap `raw` with a `fd-e` when calling the C function to get the raw fd integer, if my reading of how the rest of this file works is correct.

Reply:
Could just wrap it in a `fd-e` but *a* branch is always necessary if we want to be consistent